container_test.go

```diff
@@ -652,6 +658,17 @@ func TestInspectContainerNetwork(t *testing.T) {
 				t.Logf("%s %v", net, networkID)
 			}
 		}
+
+		var aliases []string
+		for _, net := range networks.MapKeys() {
+			if net.Interface().(string) == container.HostConfig.NetworkMode {
+				aliases = networks.MapIndex(net).FieldByName("Aliases").Interface().([]string)
+			}
+		}
+		if aliases[0] != expAlias0 || aliases[1] != expAlias1 {
```

Comment:
Can you change this check to use ``reflect.DeepEqual``? Something like:

```go
expectedAliases := []string{"testalias", "81e1bbe20b55"}
// ...
if !reflect.DeepEqual(aliases, expectedAliases) {
  t.Errorf("...", ...)
}
```

With this you can remove the variables expAlias0 and expAlias1.